src/Samples/Sample.AspNetCore/Controllers/VerifyController.cs

```diff
@@ -0,0 +1,113 @@
+{
+    public class VerifyController : Controller
+    {
+
+        private readonly ISwedbankPayClient swedbankPayClient;
+        private readonly Cart cartService;
+        private readonly PayeeInfoConfig payeeInfoOptions;
+        private readonly UrlsOptions urls;
+        private bool generatePaymentToken;
+        private bool generateReccurenceToken;
+        private string description;
+        private string userAgent;
+        private readonly StoreDbContext context;
+        private readonly SwedbankPayConnectionSettings swedbankPayConnectionSettings;
+
+
+        public VerifyController(IOptionsSnapshot<PayeeInfoConfig> payeeInfoOptionsAccessor,
+                              IOptionsSnapshot<UrlsOptions> urlsAccessor,
+                              Cart cartService,
+                              ISwedbankPayClient swedbankPayClient, StoreDbContext context, SwedbankPayConnectionSettings swedbankPayConnectionSettings)
+        {
+            this.payeeInfoOptions = payeeInfoOptionsAccessor.Value;
+            this.urls = urlsAccessor.Value;
+            this.cartService = cartService;
+            this.swedbankPayClient = swedbankPayClient;
+            this.context = context;
+            this.swedbankPayConnectionSettings = swedbankPayConnectionSettings;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+
+        public IActionResult Privacy()
+        {
+            return View();
+        }
+         
+        public async Task<CardVerify> InitiateVerify()
+        {
+            try
+            {
+                var verifyRequest = new CardPaymentVerifyRequest(Operation.Verify, new CurrencyCode("SEK"),                                                                                 
+                                                                                   description = "Test Verification", userAgent = "useragent",
+                                                                                   CultureInfo.GetCultureInfo("sv-SE"),
+                                                                                   new Urls(this.urls.HostUrls, this.urls.VerificationListUrl,
+                                                                                            this.urls.TermsOfServiceUrl, this.urls.CancelUrl,
+                                                                                            this.urls.PaymentUrl, this.urls.CallbackUrl, this.urls.LogoUrl),
+                                                                                   new PayeeInfo(this.payeeInfoOptions.PayeeId,
+                                                                                                 this.payeeInfoOptions.PayeeReference),
+                                                                                   this.payeeInfoOptions.PayeeReference,
+                                                                                   generatePaymentToken = true,
```

Comment:
Codacy found an issue: [Extract the assignment of 'generatePaymentToken' from this expression.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)